source/opt/ssa_rewrite_pass.cpp

```diff
@@ -508,20 +506,9 @@ bool SSARewriter::ApplyReplacements(Function* fp) {
   std::cerr << "\n\n";
 #endif
 
-  // Sort phi candiates by reverse postorder. Operand of a Phi may be
```

Comment:
Thank you!  This confused me.  The execution semantics of PHI nodes is that they can all execute concurrently, so sorting them did not seem necessary to me.